### DISCUSSION TITLE
E5+ update

### DISCUSSION THREAD

**Discussion by TheoBromine2022** [jon-harper/OmniBox#76] [Show and tell]
![20221129_210230](https://user-images.githubusercontent.com/111667850/205114017-6a588e92-4df0-4290-b1d3-3a6279fa2975.jpg)

Bltouch bed level test at .08mm thickness sticks to the wall. I'm calling it operational. Let me know when you want me to try those RSP 500 prints.

**Comment by jon-harper** [comment]
Sweet! I'm ready if you are!

By the way, wiring harnesses and Micro Fit 3:

I've written out and formatted a large chunk of what I've documented about making my own harnesses. Writing up stuff like this about wiring, enclosures, and filtration is under the umbrella of a different project that is less far along. Once I do some back-end work for the project's documentation site, I'll put up what I have so far.

**Comment by jon-harper** [comment]
I'm gradually adding info, but here's the framework: [https://jon-harper.github.io/clockmaker-project/](https://jon-harper.github.io/clockmaker-project/)

Much of the good stuff on standardizing pin orders is up, if rough still. Hope this helps.

**Reply by TheoBromine2022** [reply]
I have dropped off this project so hard. Sorry. I swapped my nozzle on my e5+ to a .8mm and had some serious breakthroughs with the product I'm developing. With grad school, the prototypes, the holidays and everything else... haven't had the time to mess with this. 

And I really need to, because I'm going to get more printers soon. So sometime mid January I'll get some edits going.

**Reply by jon-harper** [reply]
No need to apologize at all. Grad school on top of anything is super busy for anyone. I hope things go smoothly in setting up your farm (and in everything else!).

**Comment by TheoBromine2022** [comment]
![image](https://user-images.githubusercontent.com/111667850/209443873-b4354312-4314-45f9-b85c-40d229dc03ac.png)


Yeah, it takes up the entire e5+ plate and an entire petg roll. its an expensive prototype to say the least. (not the entire drawer, just the organizers). working on a bulk supplier

**Reply by jon-harper** [reply]
I promised myself to come back and look at this image again when I had more time to study it.

That's pretty sick.

Two ideas for you:

- You mentioned an 0.8mm nozzle. Have you tried 1 or 1.2mm? It would both further add strength and reduce print time.
- Hexagonal cutouts can reduce weight and add strength to the walls. You might even be able to thin the walls and save more material...?

**Reply by TheoBromine2022** [reply]
.8 seems to be working fine. As far as the hexagonal cutouts I'm kinda not wanting that because these inserts are going to be used by contractors and I would rather them not getting frustrated with holes snagging the pliers.

As far as strength is concerned I can stand on them and they don't break.
